pootle/apps/pootle_translationproject/views.py

```diff
@@ -25,7 +25,7 @@
 from staticpages.models import StaticPage
 
 
-ANN_COOKIE_NAME = 'project-announcements'
+ANN_COOKIE_NAME = 'sidebar-cookie'
```

Comment:
If renaming, mind doing it to something like `pootle-overview-sidebar`? Feel free to push that to master directly.